components/onboarding_tasks/onboarding_tasks_manager.tsx

```diff
@@ -101,15 +101,18 @@ export const useTasksList = () => {
     const isUserAdmin = useSelector((state: GlobalState) => isCurrentUserSystemAdmin(state));
     const isUserFirstAdmin = useSelector(isFirstAdmin);
 
+    // feature flag for ab test over the effectiveness of the onboarding workspace wizard
+    const useCaseOnboarding = useSelector(getUseCaseOnboarding);
+
     // Show this CTA if the instance is currently not licensed and has never had a trial license loaded before
     const showStartTrialTask = (isCurrentLicensed === 'false' && isPrevLicensed === 'false');
     const list: Record<string, string> = {...OnboardingTasksName};
     const pluginsPreferenceState = useSelector((state: GlobalState) => get(state, Constants.Preferences.ONBOARDING, OnboardingPreferences.USE_CASE));
     const pluginsPreference = pluginsPreferenceState && JSON.parse(pluginsPreferenceState);
-    if ((pluginsPreference && !pluginsPreference.boards) || !pluginsList.focalboard) {
+    if ((pluginsPreference && !pluginsPreference.boards) || !pluginsList.focalboard || !useCaseOnboarding) {
         delete list.BOARDS_TOUR;
     }
-    if ((pluginsPreference && !pluginsPreference.playbooks) || !pluginsList.playbooks) {
+    if ((pluginsPreference && !pluginsPreference.playbooks) || !pluginsList.playbooks || !useCaseOnboarding) {
```

Comment:
Is the check of the feature flag needed here? If they are not presented with the options because the feature flag hides the selection screen, wouldn't these just be defaulted to `false`?

Reply:
I thought the same, but the extra validation doesn't hurt. 2/5 on keeping the extra validation.